Makefile

```
@@ -35,5 +35,31 @@ update: ## Update plugin documentation and website data
+		CLAUDE_CODE_USE_VERTEX=true \
+		PROMPTFOO_PASS_RATE_THRESHOLD=$(EVAL_PASS_RATE_THRESHOLD) \
+			npx promptfoo@$(PROMPTFOO_VERSION) eval \
+			-c "$$config" \
+			$(if $(EVAL_FILTER),--filter-pattern "$(EVAL_FILTER)") \
+			$(if $(EVAL_OUTPUT),--output "$(EVAL_OUTPUT)") \
+			--repeat $(EVAL_REPEAT) \
+			--no-cache; \
+	done
+
+.PHONY: eval-security
+eval-security: ## Run security red-team evals
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)